scripts/vault-auto-organize.py

```
@@ -0,0 +1,247 @@
+
+import os
+import re
+import sys
+import subprocess
+from pathlib import Path
+from datetime import datetime
+
+VAULT = "/opt/data/obsidian/obsidian_git"
+
+def run(cmd, cwd=VAULT, check=False):
+    result = subprocess.run(cmd, shell=True, cwd=cwd, capture_output=True, text=True)
```

Ruff (0.15.12)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)